RDK-58568 : Address conversion mismatch in opendrmsession and pass sessionId by reference

ALSAMEEMA opened the pull request:
RDK-58568 : Address conversion mismatch in opendrmsession and pass sessionId by reference

Reason for change: To avoid SessionManager Session storing invalid ids
Test Procedure: Refer Ticket
Risks: Low
Priority: P1

Signed-off-by: Alsameema.A Alsameema_AhmedAnsar@Comcast.Com

aahmed878 committed:
RDK-58568 : Address conversion mismatch in opendrmsession and pass sessionId by reference

Reason for change: Allows integration of content protection service firebolt sdk
Test Procedure: Refer Ticket
Risks: Medium
Priority: P1

Signed-off-by: Alsameema.A Alsameema_AhmedAnsar@Comcast.Com

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/aamp?pullRequest=295) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/aamp?pullRequest=295) before we can accept your contribution.<br/>**19** out of **20** committers have signed the CLA.<br/><br/>:white_check_mark: anjali-syna<br/>:white_check_mark: DomSyna<br/>:white_check_mark: rajat698<br/>:white_check_mark: Vinish100<br/>:white_check_mark: anshephe<br/>:white_check_mark: p-bond<br/>:white_check_mark: pstroffolino<br/>:white_check_mark: shripadgithub<br/>:white_check_mark: nebutler963<br/>:white_check_mark: varatharajan568<br/>:white_check_mark: psiva01<br/>:white_check_mark: srikanthreddybijjam-comcast<br/>:white_check_mark: lashmintha<br/>:white_check_mark: jfagunde<br/>:white_check_mark: gavin-parry<br/>:white_check_mark: aahmed878<br/>:white_check_mark: ALSAMEEMA<br/>:white_check_mark: jameslofthouse-RED<br/>:white_check_mark: nu641001<br/>:x: VNagori<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/aamp?pullRequest=295) it.</sub>

aahmed878 committed:
RDK-58568 : Move Info Logs to MIL Logs for CPS Firebolt API Successful Invokes

Reason for change: Allows integration of content protection service firebolt sdk
Test Procedure: Refer Ticket
Risks: Low
Priority: P1

Signed-off-by: Alsameema.A Alsameema_AhmedAnsar@Comcast.Com

aahmed878 committed:
Revert "RDK-58568 : Move Info Logs to MIL Logs for CPS Firebolt API Successful Invokes"

This reverts commit 372be7f9746d94071633a1bf9880f374a2284ea7.

aahmed878 committed:
RDK-58568 : Remove Reduntant Log

Reason for change: Remove Reduntant Log
Test Procedure: Refer Ticket
Risks: Low
Priority: P1

Signed-off-by: Alsameema.A Alsameema_AhmedAnsar@Comcast.Com

ALSAMEEMA committed:
Merge branch 'dev_sprint_25_2' into feature/RDK-58568

aahmed878 committed:
RDK-58568 : Update SetDrmSessionState Log

Reason for change: Update SetDrmSessionState mismatch log
Test Procedure: Refer Ticket
Risks: Low
Priority: P1

Signed-off-by: Alsameema.A Alsameema_AhmedAnsar@Comcast.Com

pstroffolino committed:
Merge branch 'dev_sprint_25_2' into feature/RDK-58568
